### ISSUE TITLE
The html `readonly` attribute is ignored on initial load

### ISSUE THREAD

**Issue by McNetic** [autoNumeric/autoNumeric#611]
### Current behavior
Existing readonly attribute of input element is ignored (removed).

### Expected behavior
The input element having set the readonly attribute should behave the same as explicitly setting the readonly option.

### Steps to reproduce the problem
1. Use autoNumeric version `4.4.0`
2. In the browser Chrome/Firefox (current version)
3. On the Windows operating system
4. Then instantiate the autoNumeric object with the following options:
```html
<input type="text" readonly="readonly" value="1" />
```
```js
new AutoNumeric('input');
```
5. Input element is not readonly

### Link to live example (ie. [Codepen](http://codepen.io/pen/))
https://jsfiddle.net/McNetic/kgv1m680/16/

**Comment by AlexandreBonneau** [comment]
Thanks for reporting this @McNetic, I'll fix that after the next few things in my pipeline.
In the meantime, if you want to have read-only elements, use the `{ readOnly: true }` option.

**Comment by AlexandreBonneau** [comment]
This is fixed in `v4.5.1`.

The html `readonly` attribute is now correctly read and used *on the initial load*, even if the `readOnly` option is set to `false` (which is the default).
However like before, changing the `readOnly` option after the initialization will update the html attribute accordingly.

**Comment by jozzez1** [comment]
Dear all,

I am using your directive for a project of mine. In it I am using `disabled` fields instead of `readonly` because `formGroup.value` returns only values of fields that have been enabled, skipping the disabled ones.

However, much like the `readOnly` flag was ignored (removed) by the `ngAutonumeric` directive, the same goes for the `disabled`. Could you fix that in the near future?

Cheers!
